sr_world_generator/launch/create_world_template.launch

```diff
@@ -0,0 +1,23 @@
+<launch>
+    <arg name="sim" value="true"/>
+    <arg name="gui" default="true"/>
+
+    <arg name="start_home" default="true"/>
+    <arg name="scene" default="false"/>
+    <arg name="initial_z" default="0.7751"/>
+    <arg name="world" if="$(arg scene)"/>
```

Comment:
Could consider prepending the world parameter with the path so that the user doesn't have to type the full path when they specify the world file (see https://github.com/shadow-robot/sr_documentation/pull/15#discussion_r258099894) (don't have to, just a thought)

Reply:
This doesnt have default so the user HAS TO specify the world file if he wants a scene. Basically, if you want a scene, tell us which scene